b2b-app/imports/ui/forms/survey-builder/data-cache.js

```diff
@@ -51,6 +59,27 @@ const parse = (data) => {
     }),
     {}
   )
+  //add pid as a unique key for each part, so that _id is editable
```

Comment:
Add a duplicated key "pid" since "_id" is editable. if without "pid" and "_id" is changed, then there is no way to find the original question from data-cache.